[2021-08-04 13:26:09,735] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_B 2021-08-04T13:26:04.928537+00:00 [queued]>
[2021-08-04 13:26:09,908] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_B 2021-08-04T13:26:04.928537+00:00 [queued]>
[2021-08-04 13:26:09,908] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 13:26:09,909] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-04 13:26:09,909] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 13:26:09,959] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): training_model_B> on 2021-08-04T13:26:04.928537+00:00
[2021-08-04 13:26:09,973] {standard_task_runner.py:52} INFO - Started process 8175 to run task
[2021-08-04 13:26:10,059] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'training_model_B', '2021-08-04T13:26:04.928537+00:00', '--job-id', '737', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpe9n4x4ci', '--error-file', '/tmp/tmph_3myb23']
[2021-08-04 13:26:10,060] {standard_task_runner.py:77} INFO - Job 737: Subtask training_model_B
[2021-08-04 13:26:10,430] {logging_mixin.py:104} INFO - Running <TaskInstance: my_dag.training_model_B 2021-08-04T13:26:04.928537+00:00 [running]> on host 04479ea62028
[2021-08-04 13:26:10,798] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=training_model_B
AIRFLOW_CTX_EXECUTION_DATE=2021-08-04T13:26:04.928537+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-04T13:26:04.928537+00:00
[2021-08-04 13:26:10,803] {python.py:151} INFO - Done. Returned value was: 7
[2021-08-04 13:26:10,940] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=training_model_B, execution_date=20210804T132604, start_date=20210804T132609, end_date=20210804T132610
[2021-08-04 13:26:11,092] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-04 13:26:11,154] {local_task_job.py:149} INFO - Task exited with return code 0
